docs/resources/release_channel.md

```diff
@@ -191,7 +191,7 @@ Required:
 Optional:
 
 - `description` (String) description of the manual approval
-- `every_action` (Boolean) whether this approval is required for every convergence action, or just the first
+- `every_action` (Boolean) whether this approval is required for every convergence action, or just the first. This only works for runtime extensions and Terraform runners. Setting this field to true will not result in any approvals being requested for Kubernetes services.
```

Comment:
This documentation is generated from the code here:
https://github.com/prodvana/terraform-provider-prodvana/blob/b2fe48859e0ec6a4f3e7a73488e19a3957714227/internal/provider/release_channel_resource.go#L394

Could you change it there and run `go generate` ?